docs/odoo/usage/install-module.rst

```diff
@@ -63,14 +84,12 @@ install
 
 * navigate to ``Settings / Modules``
```

Comment:
Fix format https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#references-to-menu